temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using hardcoded strings. Consider using a constant or configuration file to manage such values.
```
MESSAGE = 'hello'
print(MESSAGE)
```